source/addonStore/dataManager.py

```diff
@@ -207,6 +207,16 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 
 	# Translators: A title of the dialog shown when fetching add-on data from the store fails
 	_updateFailureMessage = pgettext("addonStore", "Add-on data update failure")
+	_updateFailureMirrorSuggestion = pgettext(
+		"addonStore",
+		# Translators: A suggestion of what to do when fetching add-on data from the store fails and a metadata mirror is being used.
+		"Make sure you are connected to the internet, and the add-on store metadata mirror URL is valid.",
```

Comment:
I think "metadata" is not very user friendly

```suggestion
		"Make sure you are connected to the internet, and the add-on store mirror URL is valid.",
```

Reply:
should this include the mirror URL value?